conf/upf.json

```diff
@@ -94,10 +103,10 @@
     "slice_rate_limit_config": {
         "": "uplink policer",
         "n6_bps": 500000000,
-        "n6_burst_bytes": 3000,
+        "n6_burst_bytes": 625000,
```

Comment:
Since there is only one meter per direction, it is extremely susceptible to the issue observed in the experiment, where all packets of a batch hit the same meter.